community/CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# Contributing to SourceCred
+
+Contributing to SourceCred means any of the following:
```

Comment:
nit: there are many other ways to contribute to SourceCred (e.g. Discourse posts) -- maybe clarify that this is specific to contributing via GitHub?

Reply:
Definitely.